source/reference/minio-mc/mc-alias-set.rst

```diff
@@ -68,6 +68,22 @@ Parameters
 .. mc-cmd:: ALIAS
 
    *Required* The name to associate to the S3-compatible service.
+   Aliases are case-sensitive and must meet the following requirements:
+
+   - Contain only `ASCII <https://en.wikipedia.org/wiki/ASCII>`__ letters and numbers (``[a-zA-Z0-9]``), hyphen ``-``, or underscore ``_``.
```

Comment:
maybe something better to use here than regex syntax?

Reply:
```suggestion
   - Contain only `ASCII <https://en.wikipedia.org/wiki/ASCII>`__ lower case letters (``a-z``), upper case letters (``A-Z``), numbers (``[0-9]``), hyphen (``-``), or underscore (``_``).
```